pkg/dao/dao.go

```diff
@@ -10,11 +10,13 @@ import (
 	logging "github.com/op/go-logging"
 )
 
+//Config - confg holds etcd host and port.
```

Comment:
Add a space after the `//` not a golint check, but a zeus check. I prefer the space :smile: